stories/panel.stories.ts

```diff
@@ -150,10 +152,9 @@ storiesOf('Basics/Panel', module)
     'collapsed unless canRefine',
     withHits(
       ({ search, container }) => {
-        const breadcrumbInPanel = panel({
+        const breadcrumbInPanel = panel<BreadcrumbWidget>({
```

Comment:
```suggestion
        const breadcrumbInPanel = panel<typeof breadcrumb>({
```

can we accept this, and panel could be using something with a few layers of ReturnType?

Reply:
I just checked and `typeof breadcrumb` is `BreadcrumbWidget`. So you can already replace it without any problem, meaning we cannot use fewer layers of ReturnType even with this change. https://github.com/algolia/instantsearch.js/pull/4527/commits/d1da39bce910c1f3588cf5f7cd29a60e153ce330